src/python/pants/backend/awslambda/python/lambdex.py

```diff
@@ -11,5 +11,5 @@ class Lambdex(PythonToolBase):
     default_version = "lambdex==0.1.3"
     # TODO(John Sirois): Remove when we can upgrade to a version of lambdex with
     # https://github.com/wickman/lambdex/issues/6 fixed.
-    default_extra_requirements = ["setuptools==49.2.0"]
+    default_extra_requirements = ["setuptools==49.2.1"]
     default_entry_point = "lambdex.bin.lambdex"
```

Comment:
Can you please add `default_interpreter_constraints = ["CPython>=3.5"]`? We should have had this from the start, so you'll be fixing a bug. Because we're using setuptools > 45, we can't support Python 2.